portfolio/src/main/webapp/script.js

```diff
@@ -44,10 +44,10 @@ function presentFeedbackReceipt() {
   const userEmail = formElements[2].value;
 
   // Construct feedback receipt.
-  const receipt = `Dear ${userName},\nThank you for submitting a feedback!\n`
-                  + `We have recorded the following:\n`
-                  + `    *Message: "${userFeedback}"\n`
-                  + `    *Contact Information: ${userEmail}\n`;
+  const receipt = `Dear ${userName},\nThank you for submitting a feedback!\n` +
```

Comment:
nit: "Thank you for submitting feedback!"

Reply:
Should I remove "Dear name"?

Reply:
Oh no, I just meant removing the "a"

Reply:
Oh I see! Will do!